builder/amazon/common/run_config.go

```diff
@@ -631,6 +635,12 @@ func (c *RunConfig) Prepare(ctx *interpolate.Context) []error {
 		}
 	}
 
+	if c.Tenancy != "" && c.Tenancy != "default" {
+		if c.SpotPrice != "" {
+			errs = append(errs, fmt.Errorf("Error: Non-default tenancy cannot be used in conjunction with Spot Instances"))
```

Comment:
Why is this the case for Packer? It looks like this is possible based on AWS docs.

Reply:
Indeed this looks to be the case, removed check.